travvy/src/components/Profile.vue

```diff
@@ -73,21 +73,45 @@
 
       },
       async updateUserProfile(){
-        console.log(this.email)
-        console.log(this.age)
-        console.log(this.gender)
-        console.log(this.instagram_username)
-        console.log(this.preferred_activity_type)
-        console.log(this.errors)
-
 
       try {
+        if(this.preferred_activity_type == "Active"){
+            this.activity_id = 111
+        } else if (this.preferred_activity_type == "Food"){
+            this.activity_id = 112
+        }else if (this.preferred_activity_type == "Entertainment"){
+           this.activity_id = 113
+        } else if (this.preferred_activity_type == "Historical"){
+            this.activity_id = 114
+        } else if (this.preferred_activity_type == "Family"){
+            this.activity_id = 115
+        } else if (this.preferred_activity_type == "Nature"){
+            this.activity_id = 116
+        }
+
+        if(!this.email){
+          this.email = this.$store.state.userEmail
+        }
+        if(!this.age){
+          this.age = this.$store.state.age
+        }
+        if(!this.gender){
+          this.gender = this.$store.state.gender
+        }
+        if(!this.instagram_username){
+          this.instagram_username = this.$store.state.instagram
+        }
+        if(!this.preferred_activity_type){
+          this.preferred_activity_type = this.$store.state.activity
+        }
+
```

Comment:
I'm glad you were able to add the ability for users to have their attractions personalized on the profile page as well! Nice to give users the opportunity to change their preferred activity type.

Reply:
Thanks!